packages/@sfcc-bff/cartapi/src/api/schema/cartResolvers.js

```diff
@@ -89,9 +98,64 @@ const getBasket = async (config, context) => {
     );
 
     basket.shippingMethods = shippingMethods;
+    // get all the product ids in current basket
+    let productIds = basket.productItems
+        ? basket.productItems.map(product => product.productId).join()
+        : '';
+    // get product details for all the product items in basket
+    if (basket.productItems) {
+        let productDetails = await getProductsDetailsInfo(
```

Comment:
Why do we need to get all this extra data for the `addProductToBasket` call?

Reply:
This is in the getBasket function actually, when we add product to Basket, the product object does not contain information like image, availability, variation attributes and so on, that's why I need to make a different call to get all these info on getBasket.